src/objects/package.xml

```diff
@@ -0,0 +1,3596 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As with the layout file, there is already an existing package.xml in the src folder. That file should only have 4 changed lines for the 4 new fields.